client/gutenberg/extensions/markdown/jetpack-markdown-block.jsx

```diff
@@ -24,15 +24,17 @@ registerBlockType( 'a8c/markdown', {
 	],
 
 	icon: (
-		<svg
-			xmlns="http://www.w3.org/2000/svg"
-			class="dashicon"
-			width="20"
-			height="20"
-			viewBox="0 0 208 128"
-			stroke="currentColor"
-		>
-			<rect width="198" height="118" x="5" y="5" ry="10" stroke-width="10" fill="none" />
+		<svg xmlns="http://www.w3.org/2000/svg" width="208" height="128" viewBox="0 0 208 128">
```

Comment:
I think the width / height could be omitted here.

Reply:
Indeed 👍 Done in https://github.com/Automattic/wp-calypso/pull/27099/commits/313d49b23520fe55d65f9be0d3f75c28f10cd05c